src/extensions/paramHelper.ts

```diff
@@ -56,14 +56,15 @@ export function getParamsWithCurrentValuesAsDefaults(
  * @param envFilePath a path to an env file containing param values
  * @throws FirebaseError if an invalid env file is passed in
  */
-export async function getParams(
-  projectId: string,
-  paramSpecs: extensionsApi.Param[],
-  noninteractive: boolean = false,
-  envFilePath?: string
-): Promise<{ [key: string]: string }> {
-  if (noninteractive && !envFilePath) {
-    const paramsMessage = paramSpecs
+export async function getParams(args: {
+  projectId: string;
+  paramSpecs: extensionsApi.Param[];
+  nonInteractive?: boolean;
+  paramsEnvPath?: string;
+}): Promise<{ [key: string]: string }> {
+  let params: any;
+  if (args.nonInteractive && !args.paramsEnvPath) {
```

Comment:
nit: Should we handle the case / error out where its !args.nonInteractive and paramsEnvPath is provided?

Reply:
We do, in the `else if (args.paramsEnvPath) {` block.  Whether we're in interactive or noninteractive mode, we should read params from that file if provided.